EnglishFilter/sections/foreign.txt

```diff
@@ -925,6 +925,8 @@ fofmyanmar.com##center > a[rel="noopener noreferrer"] > img
 ! https://github.com/AdguardTeam/AdguardFilters/issues/60660
 ||zavporn.net/images/9.gif
 zavporn.net##.opt
+! https://github.com/AdguardTeam/AdguardFilters/issues/61007
+||hsex.men/view/img/*.gif
```

Comment:
Too wide. You can use Regex for these images.

Reply:
Usually it is acceptable on porn sites, ant some other.

Reply:
This part `/view/img/` seems a common, but in this case ok.